File: PYTHON/python_practice/evenodd.py

def even(n):
	c=0
	print("even num")
	for i in range(1,n+1):
		if i%2==0:
			c=c+1
			print(i)
	return c
def odd(n):
	c1=0
	print("odd num")
	for i in range(1,n+1):
		if i%2==1:
			c1=c1+1
			print(i)
	return c1

n=int(input("enter the num\n"))
x=even(n)
print("even count",x)
y=odd(n)
print("odd count",y)

File: c/arrays/evenodd.c
//Write a program in C to separate odd and even integers into separate arrays.
#include<stdio.h>
int main()
{
	int i,arr[6];
	printf("enter the elemets\n");
	for(i=0;i<6;i++)
	{
		scanf("%d",&arr[i]);
	}
	printf("even elements\n");
	for(i=0;i<6;i++)
	{
		if(arr[i]%2==0)
		{
			printf("%d\n",arr[i]);
		}
	}
	printf("odd elements\n");
	for(i=0;i<6;i++)
	{
		if(arr[i]%2!=0)
		{
			printf("%d\n",arr[i]);
		}
	}
}
